nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -41,21 +49,56 @@
 /** Unit tests for {@link com.uber.nullaway.NullAway}. */
 @RunWith(JUnit4.class)
 public class NullAwaySerializationTest extends NullAwayTestsBase {
-
-  private SerializationTestHelper serializationTestHelper;
   private String configPath;
+  private Path root;
+  private final Factory<FixDisplay> fixDisplayFactory;
+  private final Factory<ErrorDisplay> errorDisplayFactory;
+
+  enum Modes {
+    SUGGEST_FIX("fixes.tsv", SuggestedFixInfo.header()),
+    ERROR("errors.tsv", ErrorInfo.header());
+    final String fileName;
+    final String header;
+
+    Modes(String fileName, String header) {
+      this.fileName = fileName;
+      this.header = header;
+    }
+  }
+
+  public NullAwaySerializationTest() {
+    this.fixDisplayFactory =
+        values -> {
+          Preconditions.checkArgument(
+              values.length == 10,
+              "Needs exactly 10 values to create FixDisplay object but found: " + values.length);
+          // Fixes are written in Temp Directory and is not known at compile time, therefore,
+          // relative
```

Comment:
extra newline in comment

Reply:
340802a302074bc93f0147d55eb3e5b62cc8e26a